src/item.ts

```diff
@@ -115,20 +41,20 @@ interface Liveblog extends Fields {
 
 interface Review extends Fields {
     design: Design.Review;
-    body: Body;
+    body: LiveBody;
```

Comment:
Why `LiveBody` this implies Liveblog body for me

Reply:
As discussed offline, you're right this is a mistake 🙈. Fixed now.